pymeasure/adapters/protocol.py

```diff
@@ -81,11 +81,15 @@ def write_bytes(self, content):
         if self._write_buffer == to_bytes(p_write):
             assert self._read_buffer == b"", (
                 f"Unread response '{self._read_buffer}' present when writing. "
-                "Read the response or enable the property's 'check_set_errors'")
+                "Read the response; maybe a property's 'check_set_errors' is not accounted for?")
```

Comment:
This line was supposed to indicate how to fix the problem of unread messages:

Either execute "read" manually in some method, or use check_errors (with a read) in "values".

Reply:
yeah, I wanted to express flexibly that there is a mismatch. I'm not sure if "enable the property's 'check_set_errors'" is always the correct fix, could also be that the defined expected protocol is incorrect.